specs/assets_overview/02_user_interface.rst

```diff
@@ -0,0 +1,59 @@
+=====================
+User interface
+=====================
+
+=====  ===============  =========  ==========
+epic        title        author     created
+=====  ===============  =========  ==========
+`64`_  Assets overview  tschubotz  2019-01-22
+=====  ===============  =========  ==========
+
+.. _64: https://github.com/gnosis/safe/issues/64
+
+.. _`User interface`:
+
+
+#. `Main`_
+#. `User interface`_
+#. `Technical Details`_
+
+.. sectnum::
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
+
+Assets overview screen
+-------------------------
+
+- Tapping the identicon or the address brings up the receive screen.
+- Tapping "Add token" brings up the manage tokens screen.
+- Amounts should be abbreviated / truncated as follows:
+    - Cut off after the 5th decimal, no matter how many decimals there
+      are: 0.12345ETH
+    - Remove trailing zeroes, i.e. display 0.10000ETH as 0.1ETH.
+    - Use the 5 decimals up until 999.99999ETH
+    - Use 1 decimal less from 1,000.0001 until 9,999.9999ETH (Use
+      comma as thousands-separator)
+    - Use 1 decimal less from 10,000.001 until 99,999.999ETH
```

Comment:
no space between the number and token symbol?

I assume all token amounts have the same symbol-digit order, right? Unlike USD: `$33.00` and EUR: `33,00€`

Reply:
Also, we always show the token symbol, right? The screen example in this document does not show a token symbol after the numbers.

Reply:
> no space between the number and token symbol?
> I assume all token amounts have the same symbol-digit order, right? Unlike USD: $33.00 and EUR: 33,00€
> Also, we always show the token symbol, right? The screen example in this document does not show a token symbol after the numbers.

Actually, on this screen we don't even display the token symbol. Will remove it.